11LessonEleven/in-class-project/index.html

```diff
@@ -1,10 +1,40 @@
-<!DOCTYPE html>
-<html>
-<head>
-  <title>Intel</title>
-  <link rel="stylesheet" href="css/style.css">
-</head>
-<body>
-
-</body>
-</html>
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>lesson 11 in class</title>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="css/css-reset.css" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <main class="main-start">
+        <div class="welcome">
+            <h1>Welcome to our site</h1>
+        </div>
+    </main>
+    <section class="box-clearfix">
+        <div class="first-box">
+            <div class="foo-blue"></div>
+            <h1>Title</h1>
+            <p>Direct trade cold-pressed four dollar toast tacos, taxidermy XOXO whatever aesthetic green juice. Art party banjo DIY gastropub semiotics, authentic raw denim organic austin messenger bag bitters before they sold out neutra hashtag </p>
+        </div>
+        <div class="second-box ">
+            <div class="foo-red"></div>
+            <h1>Title</h1>
+            <p>Lo-fi intelligentsia mixtape chambray, migas craft beer next level roof party disrupt microdosing jean shorts gluten-free williamsburg pabst. Hoodie post-ironic heirloom beard. Wolf bitters chicharrones pork belly bushwick</p>
+        </div>
+        <div class="third-box ">
+            <div class="foo-teal"></div>
+            <h1>Title</h1>
+            <p>Pabst swag chambray health goth deep church-key, chicharrones occupy craft beer williamsburg keffiyeh helvetica thundercats brooklyn. XOXO tote bag polaroid next level Master cleanse microdosing pabst</p>
+        </div>
+        <div class="clearfix" </div>
+    </section>
+
+</body>
+
+</html>
```

Comment:
<img width="1032" alt="screen shot 2016-09-13 at 5 35 58 pm" src="https://cloud.githubusercontent.com/assets/12698279/18493901/93afb306-79d8-11e6-9c70-2ce399a8231a.png">
Looks like the center TITLE div isn't quite centered, there is more space to its right than left. Make this change, let me know and i can bump your grade up.